tests/unit/dependabot_alerts/core_test.py

```diff
@@ -1,6 +1,3 @@
-from dependabot_alerts.core import hello_world
-
-
-class TestHelloWorld:
+class TestIt:
     def test_it(self):
-        assert hello_world() == "Hello, world!"
+        pass
```

Comment:
I don't think we want to bother writing any actual unit tests for `dependabot-alerts`'s core at this point, so just replacing the auto-generated test (which depends on the auto-generated `hello_world()` function that I deleted) with a placeholder test here.